ansible/roles/dev-desktop/files/scripts/setup_rust.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Enable strict mode for Bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+username=$(id -u -n)
+gh_name=${username#"gh-"}
+
+set -x
+
+if [[ ! -d "rust" ]]; then
+  # Using https instead of git urls because vscode only handles login on push/pull
+  git clone "https://github.com/${gh_name}/rust.git"
+fi
+
+pushd rust
+
+if ! git remote | grep upstream; then
+  git remote add upstream https://github.com/rust-lang/rust.git
+fi
+
+git fetch upstream
+git checkout upstream/master
+popd
+
+set_defaults.sh
+link_rust.sh
```

Comment:
@jyn514 Since we removed `./x.py build` from `link_rust.sh`, should we put it in here? Without it, calling `link_rust.sh` is kinda useless.